scripts/benchmark_migration.py

```diff
@@ -177,18 +178,23 @@ def main(
         for model in models:
             missing = min_entities - model_rows[model]
             if missing > 0:
+                entities: List[Model] = []
+                bar = ChargingBar("Processing", max=missing)
```

Comment:
How does the print in 183 interact with the bar? I think it might be better to reverse the order of 182,183.

Can you post a screencast of the progress bar in action? :)

Reply:
I just switched and pushed up, it does look a lot better. Putting a screencast in the main portion.